packages/repository-json-schema/test/integration/build-schema.integration.ts

```diff
@@ -56,8 +60,12 @@ describe('build-schema', () => {
           bar: number;
         }
 
-        expect(modelToJsonSchema(Empty)).to.eql({});
-        expect(modelToJsonSchema(NoModelMeta)).to.eql({});
+        const emptyJson = modelToJsonSchema(Empty);
+        const noModelMetaJson = modelToJsonSchema(NoModelMeta);
+        expect(emptyJson).to.eql({});
```

Comment:
This seems redundant. Won't `expectValidJsonSchema` check if it's `{}` or not?

Reply:
I guess it is redundant. On the other hand though, what if someday they decide `{}` is not a valid JSON Schema? In that case, we'd want this to test to fail while still maintaining that `emptyJson` is an empty object and that `{}` is not a valid JSON Schema

Reply:
Unlikely, but I'm ok with the redundancy.